cmd/statusd/debug/commands.go

```diff
@@ -0,0 +1,149 @@
+package debug
+
+import (
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"github.com/status-im/status-go/geth/api"
+	"github.com/status-im/status-go/geth/params"
+)
+
+// command contains the result of a parsed command line and
+// is able to execute it on the command set.
+type command struct {
+	funcName string
+	args     []interface{}
+}
+
+// newCommand parses a command line and returns the command
+// for further usage.
+func newCommand(commandLine string) (*command, error) {
+	expr, err := parser.ParseExpr(commandLine)
+	if err != nil {
+		return nil, err
+	}
+	switch expr := expr.(type) {
+	case *ast.CallExpr:
+		return &command{
+			funcName: (expr.Fun.(*ast.Ident)).Name,
+			args:     exprsToArgs(expr.Args),
+		}, nil
+	default:
+		return nil, fmt.Errorf("invalid command line: %q", commandLine)
+	}
+}
+
+// exeecute calls the method on the passed command set value.
+func (c *command) execute(csv reflect.Value) (replies []string, err error) {
```

Comment:
What does `csv` stand for?

Reply:
It's the command set value, the `reflect` value of the `commandSet` instance. Gave it a full name.